File: 백준/Bronze/10810. 공 넣기/공 넣기.cc
#include<iostream>
using namespace std;

int arr[105];

int main()
{
	int n, m;
	cin >> n >> m;

	while (m--) {
		int i, j, k;
		cin >> i >> j >> k;

		for (int s = i; s <= j; s++) arr[s] = k;
	}

	for (int i = 1; i <= n; i++) cout << arr[i] << ' ';

	return 0;
}

File: 백준/Bronze/10810. 공 넣기/공 넣기.java
import java.util.Scanner;

class Main {
	public static void main(String[] args) throws Exception {
		Scanner scanner = new Scanner(System.in);
		int n = scanner.nextInt();
		int m = scanner.nextInt();
		int [] arr = new int[n];
		for(int i = 0; i<m; i++){
			int a = scanner.nextInt();
			int b = scanner.nextInt();
			int k = scanner.nextInt();
			for(int j = a-1; j < b; j++){
				arr[j] = k;
			}
		}
		for(int i = 0; i<n; i++){
			System.out.print(arr[i] + " ");
		}
	}
}